index.html

```diff
@@ -103,65 +109,60 @@ <h2>Version Control comes to rescue</h2>
                 <h2>Why Git?</h2>
                 <section>
                     <p>
-                    <ul>
-                        <li class="fragment">You can use git even when you're offline</li>
-                        <li class="fragment">Code revisions (commit) that you made would be available to only you locally
-                            unless you push to remote repo.
-                        </li>
-                        <li class="fragment">Create branches, work locally, isolate your work and merge seamlessly.</li>
-                        <li class="fragment">It takes snapshots of changes and treat files as stream of snapshots over
-                            time.
-                        </li>
-                    </ul>
+                        <ul>
+                            <li class="fragment">It's free and open-source.</li>
+                            <li class="fragment">You can use git even when you're offline</li>
+                            <li class="fragment">Code revisions (commit) that you made would be available to only you locally unless you push to remote repo.
+                            </li>
+                            <li class="fragment">Create and support parallel branches, work locally, isolate your work and merge seamlessly to collaborate.</li>
+                            <li class="fragment">It takes snapshots of changes and treat files as stream of snapshots over time.
+                            </li>
+                        </ul>
                     </p>
                 </section>
                 <section>
                     <p>
-                    <ul>
-                        <li class="fragment">It compresses the .git contents</li>
-                        <li class="fragment">It has integrity</li>
-                        <li class="fragment">The mechanism that Git uses for this checksumming is called a SHA-1 hash. This
-                            is a 40-character string composed of hexadecimal characters (0–9 and a–f) and calculated based
-                            on the contents of a file or directory structure in Git. A SHA-1 hash looks something like this:
-                            <pre><code class="hljs" data-trim
-                                       contenteditable>24b9da6552252987aa493b52f8696cd6d3b00373</code></pre>
-                        </li>
-                    </ul>
+                        <ul>
+                            <li class="fragment">It compresses the .git contents</li>
+                            <li class="fragment">It has integrity</li>
+                            <li class="fragment">The mechanism that Git uses for this checksumming is called a SHA-1 hash. This is a 40-character string composed of hexadecimal characters (0–9 and a–f) and calculated based on the contents of a file or directory structure
```

Comment:
Trim these lines as these were before, and do this in other parts too which you've touched. Although IDEs fix this, but let's ensure that even if I'm opening it on a simple text editor, `nano`, `vim`, `TextEdit` or whatever, I could see everything without doing a horizontal scroll.

Reply:
So should we enforce 79 char per line? because indentation will be indentation.

Reply:
Yeah for long lines not for others. Can go to 120 chars as well. There is no hard limit. But I just don't want something really long, running entire 2 screens.

Reply:
Gotcha